### PR TITLE
Create shell script for wilms-06 workflow

### PR DESCRIPTION
**Pull request by sjspielman**
Towards #816 

This PR adds a new shell script to run the `cell-type-wilms-tumor-06` workflow. This does NOT contain any of the active changes in #814, since I wanted to make sure a directly comparable version of this script was in place before changing up the steps. As such, this is basically a "shell version" of `00_run_workflow.R`.

Other changes include:
- updating the workflow yaml file to use this script, and run on this branch
- updating the module README to reference this script instead of the old one. Note this file got got by precommit, hence whitespace diffs.

EDIT: Link to workflow R script for comparison during review: https://github.com/AlexsLemonade/OpenScPCA-analysis/blob/feature/wilms-tumor-06-azimuth/analyses/cell-type-wilms-tumor-06/00_run_workflow.R

### CODE REVIEW

File: analyses/cell-type-wilms-tumor-06/00_run_workflow.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+data_dir="../../data/current"
+notebook_template_dir="notebook_template"
+notebook_output_dir="notebook"
+
+
+# Download and create the fetal kidney reference (Stewart et al)
+Rscript scripts/download-and-create-fetal-kidney-ref.R
+
+# Build the gene position file reference for infercnv
+Rscript scripts/06a_build-geneposition.R
```
Comment (jashapiro) [inline analyses/cell-type-wilms-tumor-06/00_run_workflow.sh:30]: It is confusing to have this out of numerical order. Do we want to rename the file or move this step?

File: analyses/cell-type-wilms-tumor-06/README.md [hunk 2]
```diff
@@ -137,11 +137,11 @@ We gather from the litterature marker genes and specific genomic alterations tha
 
 ## Clustering and label transfer from fetal references
 
-R Script to be rendered : `00_run_workflow.R`
+R Script to be rendered : `00_run_workflow.sh`
```
Comment (jashapiro) [inline analyses/cell-type-wilms-tumor-06/README.md:140]:
```suggestion
Bash script to be run : `00_run_workflow.sh`
```

**Review by jashapiro — APPROVED** [review]
Looks good as a general translation of the R script.

File: analyses/cell-type-wilms-tumor-06/00_run_workflow.sh (hunk 1 above)
Comment (sjspielman) [reply analyses/cell-type-wilms-tumor-06/00_run_workflow.sh:30]: I agree. I moved it at one point, and then in the spirit of keeping things as they were, I unmoved it. But you see it too, so yes let's move it.

**Pull request merged**